src/org/sosy_lab/java_smt/solvers/boolector/BtorJNI.java

```diff
@@ -1,3 +1,11 @@
+// This file is part of JavaSMT,
+// JavaSMT is an API wrapper for a collection of SMT solvers.:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
 /* ----------------------------------------------------------------------------
  * This file was automatically generated by SWIG (http://www.swig.org).
```

Comment:
What file is this based on? Generated code typically needs to be considered derived from the input of the generator (e.g., a grammar in case of parser generators). Depending on the license of the original file, we might need to keep the original copyright and license notices.